subiquity/models/installpath.py

```diff
@@ -28,23 +28,38 @@ class InstallpathModel(object):
 
     path = 'ubuntu'
     # update() is not run, upon selecting the default choice...
-    source = '/media/filesystem'
     curtin = {}
+    cmdline_sources = None
+
+    def __init__(self, sources=None):
+        self.cmdline_sources = sources
+        if sources:
+            self.path = 'cmdline'
 
     @property
     def paths(self):
-        return [
+        cmdline = []
+        if self.cmdline_sources:
+            cmdline = [(_('Command Line Source'), 'cmdline')]
```

Comment:
Install from Command Line Source?

Reply:
changed that to 'Install from cli provided sources'.

Reply:
That'll need word smithing as well; though it's not shown by default.  Maybe:

Install from Custom Source?

We can ask design.